### DISCUSSION TITLE
10월 24일 주간회의

### DISCUSSION THREAD

**Discussion by her0807** [woowacourse/prolog#1090] [General]
한주간 고생하셨습니다. 😀 작업 내역 공유 스레드 미리 만들어둡니다!!!

**Comment by her0807** [comment]
## 로드맵 퀴즈 구현 완료 했습니다. 
- 루키와 생성한 roadmap 브런치에서  `키워드와 퀴즈`에 대한 리뷰 진행한 뒤, 병합하여 최종 PR 보내도록 하겠습니다. 
- https://github.com/woowacourse/prolog/pull/1088

**Comment by hanull** [comment]
### 이미지 삽입 기능 구현 완료
- 기존 aws-sdk version2 로 작성된 코드에서 credential 이슈 발생하여 version1을 사용하도록 변경
- 중복된 이름 생기지 않도록 uuid 적용
- dev 배포 후, 정상 동작 테스트 완료
https://github.com/woowacourse/prolog/pull/1086

**Comment by euijinkk** [comment]
## 로드맵 페이지 기능 구현 완료했습니다. (with @wonsss)

- 미래에 기능이 확장될 것이므로... 코드 깔끔하게 리팩터링 해두었습니다.

<img width="500" alt="image" src="https://user-images.githubusercontent.com/24906022/197450758-616e2192-3253-461f-b606-cec6c8db227a.png">
<img width="500" alt="image" src="https://user-images.githubusercontent.com/24906022/197450797-d915b12b-e026-49a0-a883-f1762f14b67a.png">

### 이슈사항
develop이랑 main이랑 깨져서 main에서 판 PR을 develop에 올릴 수가 없어요. develop 파기하고 올리면 되는지... 아직 브랜치전략이 익숙하지 않네요. 전체적으로도 혼용되고 있는 것 같아요!

**Comment by wishoon** [comment]
## 로드맵 키워드 구현 현재 진행중

현재 작업 90% 정도 완료되어서 오늘 저녁 중으로 PR 요쳥 예정입니다. 위의 수달이 언급한 roadmap 브런치에서 키워드와 퀴즈에 대한 리뷰 진행한 뒤, 병합하여 최종 PR 보내도록 하겠습니다. (내년 프롤로그 Ver 2의 중축?이 될 PR이라 빡센 리뷰 부탁드립니다 ㅎㅎ..)
* https://github.com/woowacourse/prolog/issues/1087

## 댓글 PR ISSUE

댓글 관련 PR도 오늘 중으로 Merge 할 예정입니다. 관련된 Issues가 있는데 확인해주시면 좋을 것 같습니다.
* 현재 PR Merge가 Develop에 된 PR도 있고, Main에 되어 있는 PR도 있습니다. 충돌이 날 것으로 보이는데 Git Branch 전략에 대해서 한번 더 확인이 필요할 것 같습니다.

**Comment by JUDONGHYEOK** [comment]
### 권한 (등업 신청) 회의 및 프론트엔드 구현 완료

회의 결과 이슈
#937
<img width="223" alt="image" src="https://user-images.githubusercontent.com/65863017/197452876-4bdbb975-abb9-447e-acaa-6110cd71dab2.png">
- 해당 UI 구현 및 API 모킹

**Comment by Yboyu0u** [comment]
### java version9로 문제 발생하여 pr revert 후 version8로 다시 다운 
문서화 테스트 개선 dev에 merge이후, java version 9로 변경했던 것이 문제가 되어 9에 종속된 코드 수정 후, 재 pr후merge 완료. 
관련 pr: https://github.com/woowacourse/prolog/pull/1091

### 브리핑 내용 
- 저번 주 할당 받은 이슈인 cloudfront적용에 대해선 아직 진행사항 없음 
관련 이슈: https://github.com/woowacourse/prolog/issues/1075

**Comment by al-bur** [comment]
- 임시저장 리뷰에 대한 코드 리팩토링 진행중

- 어드민 기능 일정 계획 수립
https://github.com/woowacourse/prolog/issues/1071#issuecomment-1288422667
